docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+# source the setup-environment file
+#  source setup-environment
+#
+# then launch this with:
+#
+#   $ docker-compose build esp32-builder
+#   $ docker-compose run --rm esp32-builder
+
+version: '3'
+
+services:
+  esp32-builder:
+    build: .
+    image: jci/esp32-builder:latest
+    environment:
+      - CCACHE_DIR=/ccache
+    devices:
+      - "/dev/ttyUSB0:/dev/ttyUSB0"
```

Comment:
Out of curiosity, does Docker for Mac support sharing of USB devices? Last I used it, it wasn't supported but maybe things have changed.